svc-bie-kafka/src/main/resources/application-dev.yaml

```diff
@@ -1,24 +1,23 @@
 spring:
   kafka:
-    bootstrap-servers: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:kafka.dev.bip.va.gov:443}"
+    bootstrap-servers: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:bip-kafka.tst.bip.va.gov:443}"
```

Comment:
If we are updating urls there are a probably some docs that we should plan to synchronize with this change.